app/models/user.rb

```diff
@@ -13,7 +15,10 @@ class User < ActiveRecord::Base
   validates :password, confirmation: true, presence: true,
             length: { minimum: 3 }
   validates :password_confirmation, presence: true
-  validates :email, uniqueness: true, presence: true
+  validates :email, uniqueness: true, presence: true,
+            format: { :with => /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/ }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.